presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -321,30 +319,27 @@ There are four selectors, that define which queries run in which resource group:
 
 * The first selector matches queries from ``bob`` and places them in the admin group.
 
-* The second selector matches queries from ``admin`` user group and places them in the admin group.
-
-* The third selector matches all data definition (DDL) queries from a source name that includes ``pipeline``
+* The second selector matches all data definition (DDL) queries from a source name that includes ``pipeline``
   and places them in the ``global.data_definition`` group. This could help reduce queue times for this
   class of queries, since they are expected to be fast.
 
-* The fourth selector matches queries from a source name that includes ``pipeline``, and places them in a
+* The third selector matches queries from a source name that includes ``pipeline``, and places them in a
   dynamically-created per-user pipeline group under the ``global.pipeline`` group.
 
-* The fifth selector matches queries that come from BI tools which have a source matching the regular
-  expression ``jdbc#(?<toolname>.*)`` and have client provided tags that are a superset of ``hipri``.
-  These are placed in a dynamically-created sub-group under the ``global.adhoc`` group.
-  The dynamic sub-groups are created based on the values of named variables ``toolname`` and ``user``.
-  The values are derived from the source regular expression and the query user respectively.
-  Consider a query with a source ``jdbc#powerfulbi``, user ``kayla``, and client tags ``hipri`` and ``fast``.
-  This query is routed to the ``global.adhoc.bi-powerfulbi.kayla`` resource group.
+* The fourth selector matches queries that come from BI tools which have a source matching the regular
+  expression ``jdbc#(?<toolname>.*)``, and have client provided tags that are a superset of ``hi-pri``.
+  These are placed in a dynamically-created sub-group under the ``global.pipeline.tools`` group. The dynamic
+  sub-group is created based on the named variable ``toolname``, which is extracted from the
+  regular expression for source. Consider a query with a source ``jdbc#powerfulbi``, user ``kayla``, and
```

Comment:
```suggestion
  regular expression for source. 
  
  Consider a query with a source ``jdbc#powerfulbi``, user ``kayla``, and
```
Suggest breaking up this dense paragraph with a blank line before the example for readability. Please keep the example in the unordered list item for "The fourth selector". 

I'm including screenshots of the paragraph as is (without the line break) and with this suggestion (as built locally) to demonstrate what I am suggesting. 
<img width="1037" alt="Screenshot 2024-01-30 at 9 57 22 AM" src="https://github.com/prestodb/presto/assets/7013443/b26c7c1e-3054-4f69-9a65-8c1053e38cbb">
<img width="1037" alt="Screenshot 2024-01-30 at 9 57 39 AM" src="https://github.com/prestodb/presto/assets/7013443/efad6ff5-347a-46f0-925b-2ac4ac91eaf9">

Reply:
done